### DISCUSSION TITLE
Slint on PineTime (Nordic Semiconductor nRF52832)

### DISCUSSION THREAD

**Discussion by realroot2185** [slint-ui/slint#7084] [Q&A]
Is it possible to use slint there?

**Comment by tronical** [comment]
I don’t have this device, so I can’t say for sure. The flash storage looks good, but 64/32 KiB RAM is rather little - that would be my main concern (tool chain etc aside).

**Comment by Vadoola** [comment]
I'm pretty sure I remember seeing someone working on an app for PineTime with Slint...I'll have to see if I can find it. Not sure what the status is or how well it works.

Edit: never mind I think this is what I was thinking of which isn't the PineTime: https://github.com/igiona/rs-watch

**Comment by realroot2185** [comment]
Is there some guide to how use slint for embedded?
I can check that repo.
